### PR TITLE
Add phase tap to twt modification

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/components/dialogs/network-modifications/two-windings-transformer/creation/tap-changer-pane/phase-tap-changer-pane/phase-tap-changer-pane-steps.js [hunk 1]
```diff
@@ -15,13 +15,17 @@ import {
...
 import { useIntl } from 'react-intl';
 import TapChangerSteps from '../tap-changer-steps';
 import { parseIntData } from '../../../../../dialogUtils';
 import { PHASE_TAP } from '../../two-windings-transformer-creation-dialog';
 
-const PhaseTapChangerPaneSteps = ({ disabled }) => {
+const PhaseTapChangerPaneSteps = ({
+    disabled,
+    twtToModify,
+    modification = false,
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/network-modifications/two-windings-transformer/creation/tap-changer-pane/phase-tap-changer-pane/phase-tap-changer-pane-steps.js:27]: rename "modification" => "isModifcation" or "isTwtModification" be better

File: src/components/dialogs/network-modifications/two-windings-transformer/creation/tap-changer-pane/phase-tap-changer-pane/phase-tap-changer-pane-utils.js [hunk 2]
```diff
@@ -39,66 +41,78 @@ import {
...
     REGULATION_TYPES,
     SIDE,
 } from 'components/network/constants';
+import {
+    getRegulatingTerminalEmptyFormData,
+    getRegulatingTerminalFormData,
+} from 'components/dialogs/regulating-terminal/regulating-terminal-form-utils';
 
-const phaseTapChangerValidationSchema = (id) => ({
+const phaseTapChangerValidationSchema = (modification, id) => ({
```
Comment (Walid-Sahnoun) [inline src/components/dialogs/network-modifications/two-windings-transformer/creation/tap-changer-pane/phase-tap-changer-pane/phase-tap-changer-pane-utils.js:54]: same here

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: minor changes requested

tests:
- I got the same "updateResource()" issue in incremental mode, then I had to use locally this [PR](https://github.com/powsybl/powsybl-network-store/pull/340)
(other workaround: rebuild the whole node to make sure the resource is flushed to db).
- I think the deadband adornement (MV) is bad. To be checked with PO. This is the same in the Ratio tab.
- issue. test case:
	- create a 2wt modification on unknown id "ttt"
	- enable phase tap changer
	- set "current limiter" regulation, local, 20A, 0.2 (deadband), Side2
	- save/open, the content is not good (regulation became Remote)

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

**Review by archendel — APPROVED** [review]

**Pull request merged**